packages/react-components/react-dialog/src/contexts/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const MEDIA_QUERY_BREAKPOINT_SELECTOR = '@media screen and (max-width: 480px)';
```

Comment:
none of these constants are used in this PR

Reply:
I did that actually, I guess some small things slipped 🙈